agent/taskresource/logrouter/logrouter_unimplemented.go

```diff
@@ -0,0 +1,135 @@
+// +build !linux
+
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package logrouter
+
+import (
+	"errors"
+	"time"
+
+	"github.com/aws/amazon-ecs-agent/agent/api/task/status"
+	"github.com/aws/amazon-ecs-agent/agent/taskresource"
+	resourcestatus "github.com/aws/amazon-ecs-agent/agent/taskresource/status"
+)
+
+const (
+	// ResourceName is the name of the log router resource.
+	ResourceName = "logrouter"
+)
+
+// LogRouterResource represents the log router resource.
+type LogRouterResource struct{}
```

Comment:
Just for my own understanding... what's the point of this "unimplemented" LogRouterResource?

Reply:
the current implementation of LogRouterResource is very specific to Linux so it's done in logrouter_unix.go with a linux build tag (at the first line). This file has a "!linux" build tag so the code can compile in all platforms without having to restrict the reference of LogRouterResource in linux specific place as well (which probably isn't possible without refactoring existing agent code)